libs/lb-fs/src/mount.rs

```diff
@@ -71,5 +71,6 @@ pub async fn umount() {
 
 #[cfg(target_os = "windows")]
 pub async fn umount() {
+    info!("todo");
```

Comment:
info is an unused important in the file when using windows. Stuck this in here since the other unmount functions used info in a similar place.

Perhaps finishing this todo is a next action for me?

Reply:
fwiw the `todo!()` will `eprintln` a panic message. The underlying windows library we use for this is this lib: https://github.com/xetdata/nfsserve

And it only supports windows pro, do you have windows pro? Have you also messed around with the `fs` command? I think there's going to be a longer term windows-centric way to do `fs` on windows.

If you find this body of work interesting we can talk about areas of inquiry. I would be curious about exploring something like this: https://learn.microsoft.com/en-us/windows/win32/projfs/projected-file-system

could be other approaches as well